src/test/java/redis/clients/jedis/tests/commands/ClientIdCommandsTest.java

```diff
@@ -0,0 +1,73 @@
+package redis.clients.jedis.tests.commands;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.Jedis;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests related to clientId
+ */
+public class ClientIdCommandsTest extends JedisCommandTestBase {
+    private final String clientName = "fancy_jedis_name";
+    private final String clientName2 = "fancy_jedis_another_name";
+
+
+    protected Jedis client;
+    private Jedis client2;
```

Comment:
This is used only in `clientIdMultipleConnection()`. So create this in that method. Also declare `clientName2` there.